library/core/src/main/java/com/google/android/exoplayer2/text/ttml/TtmlDecoder.java

```diff
@@ -755,6 +764,27 @@ private static void parseFontSize(String expression, TtmlStyle out) throws
     }
   }
 
+  private static void parseShear(String expression, TtmlStyle out) throws
```

Comment:
Is it possible to switch this to return `float` instead of taking an output parameter?